site/content/en/blog/releases/1.20.0-rc.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "1.20.0-rc - simple game server version to 0.6"
+linkTitle: "1.20.0-rc"
+date: "2022-01-04"
+---
+
+
+This is the 1.20.0-rc release of Agones.
+
+This release brings us  the simple game server version 0.6 and grpc client and server on same version
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.20.0-rc" >README</a> for details on features, installation and usage.
```

Comment:
This will need `data-proofer-ignore` - see the template I linked in the last comment. (Since the branch doesn't exist yet).